resources/lib/vrtplayer/metadatacreator.py

```diff
@@ -205,6 +223,11 @@ def get_video_dict(self):
         if self.url:
             video_dict['path'] = self.url
 
+        if self.whatsonid:
+            video_dict['dbid'] = self.whatsonid
+        #if self.videoid:
+        #    video_dict['dbid'] = self.videoid
+
```

Comment:
Warning, whatsonid is not always present in the api: https://vrtnu-api.vrt.be/search?i=video&facets[url]=//www.vrt.be/vrtnu/a-z/fc-de-kampioenen/3/f-c--de-kampioenen-s3a3/
https://vrtnu-api.vrt.be/search?i=video&facets[url]=//www.vrt.be/vrtnu/a-z/postbus-x/1/postbus-x-s1a8/

videoId on the other hand is always present. The only problem is converting this UUID to an unique integer.

Reply:
Euh, don't tell me I released this with those bits :-(
That was not the plan. I am pretty sure I stashed my local changes before making the ZIP file.

Reply:
Aha, it's only in this PR, pfhew.
So yes, this was some test-code that probably cause all files to be playing together.
Which I think means Kodi is assuming this is the same content, so I think that will fix our dependency on the URL (that could change in the future).

Reply:
Fixed.